modules/ocf_desktop/files/xsession/xflock4

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+#  xfce4
+#
+#  Copyright (C) 1999, 2003 Olivier Fourdan (fourdan@xfce.org)
+#  Copyright (C) 2011       Guido Berhoerster (guido+xfce.org@berhoerster.name)
+#  Copyright (C) 2011       Jarno Suni (8@iki.fi)
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+
+PATH=/bin:/usr/bin
+export PATH
+
+# Lock by xscreensaver or gnome-screensaver, if a respective daemon is running
+for lock_cmd in \
+    "xscreensaver-command -lock" \
+    "light-locker-command --lock" \
+    "gnome-screensaver-command --lock"
+do
+    /usr/local/bin/staff-logout &
+    $lock_cmd >/dev/null 2>&1 && exit
+done
```

Comment:
no it does not. if the only one you have installed is gnome-screensaver, it will launch 3 copies of our script in the background.